template.yaml

```diff
@@ -343,8 +353,8 @@ Resources:
           Environment:
             - Name: API_BASE_URL
               Value: !FindInMap [EnvironmentVariables, !Ref Environment, APIBASEURL ]
-            - Name: EXTERNAL_WEBSITE_HOST
-              Value: !FindInMap [EnvironmentVariables, !Ref Environment, EXTERNALWEBSITEHOST ]
+            - Name: FRONT_END_CUSTOM_DOMAIN
```

Comment:
Can you check with an FE engineer that this is the right env var to pass the custom domain name in to the container on?

Reply:
now replaced with exerternal website host